sansa-ml-spark/src/test/scala/net/sansa_stack/ml/spark/wordnet/DistanceWordNetSimilarityMeasureTests.scala

```diff
@@ -1,8 +1,9 @@
-package net.sansa_stack.ml.spark.nlp.wordnet
+package net.sansa_stack.ml.spark.wordnet
 
 import com.holdenkarau.spark.testing.DataFrameSuiteBase
-import net.sf.extjwnl.data._
+import net.sansa_stack.ml.common.nlp.wordnet.{WordNet, WordNetSimilarity}
 import org.scalatest.FunSuite
+import net.sf.extjwnl.data._
 
 class DistanceWordNetSimilarityMeasureTests extends FunSuite with DataFrameSuiteBase {
```

Comment:
I think you should consider moving unit-test to the common module as well since none of them depends on Spark (even you mention it on the comment). Instead of using DataFrameSuiteBase which is an extension to FunSuite for including SparkSession, there is no need in our case to use it. So, just use `FunSuite` and all the unit-test for the Wordnet can be moved to the common module as well.
Do the same for all.

Reply:
It gave me an error without DataFrameSuiteBase. I will give it another try maybe I find a workaround